src/NuGet.Clients/NuGet.PackageManagement.UI/PackageManagerToolWindowPane.cs

```diff
@@ -13,14 +14,16 @@ namespace NuGet.PackageManagement.UI
     public class PackageManagerToolWindowPane : ToolWindowPane, IVsWindowFrameNotify3
     {
         private PackageManagerControl _content;
+        private string _projectGuid;
```

Comment:
Can you eliminate this and just use a read-only property?
```C#
public string ProjectGuid { get; }
```
If no, make this `readonly`.

Reply:
fixed in next iteration.